config.go

```diff
@@ -913,9 +912,14 @@ func loadConfig(ctx context.Context) (*config, []string, error) {
 	}
 
 	// AllowedSplitTxMaxValue sanity check
-	if cfg.TBOpts.SplitTx > maxAllowedSplitTxValue {
-		str := "%s: splittx value must not exceed %v"
-		err := fmt.Errorf(str, funcName, maxAllowedSplitTxValue)
+	// While sstx transactions with many inputs can be created,
+	// the transactions end up being innefficient in size and fee
+	// because of the current sstx output ratio requirements.
+	// For this reason, split transactions will always be enforced for
```

Comment:
fee efficiency is not the only consideration.   A non-split funded sstx would be good to avoid an extra block before the ticket is mined.  I don't think the split and the sstx can be mined in the same block.  Is that correct?

Also "innefficient" -> inefficient

Reply:
See my comment below where I have explained it in detail.